### PR TITLE
First draft of seed scripting completed

### PR DESCRIPTION
**Pull request by theofficetroll**
https://trello.com/c/Neh5Z5E2

### CODE REVIEW

File: database/index.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+});
+
+var navbarSchema = mongoose.Schema ({
+  bundleId: Number,
+  countDown: Number
+});
+
+const Navbar = mongoose.model('Navbar', navbarSchema);
+
+let navbarSave = (data) => {
```
Comment (gektin) [inline database/index.js:24]: You're mixing your variable declaration with `var` and `const`/`let` here, so I think you should think through how you want to be declaring variables throughout. I believe that the airbnb style guide prefers the ES6 way. Consider also whether a `const` would be more appropriate for the function declaration on line 24 here.

File: database/index.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    return {
+      bundleId: data.bundleId,
+      countDown: data.countDown
+    }
+  })
+  return Navbar.insertMany (navbarData)
+  .then (result => {
+    console.log ('saved to database');
+  })
+  .catch (err => console.log ('failed to save to database', err));
```
Comment (gektin) [inline database/index.js:35]: Another consistency point here. If you look at lines 14 and 22 of this file, you're invoking a function with the function name and no space between that and the invoking parentheses, whereas here (as well as on lines 17 and 25), you have a space between the function name and the invocation. Pick one and stick with it. I would lean toward having no spaces.

File: database/seed.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+seedGen = function () {
+  var bundleId, countDown;
+  var timerData = [];
+  for (let i = 1; i < 100; i++) {
+    bundleId = i;
+    timer = Number(Math.floor (Math.random() * 13) + 1);
+    countDown = timer;  // countdown in days, may adjust to be more granular later
+    timerData.push ({
+      bundleId : bundleId,
+      countDown : countDown
```
Comment (gektin) [inline database/seed.js:16]: Recommend using ES6 here for readability per the airbnb style guide.

File: database/seed.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    })
+  }
+  return timerData;
+};
```
Comment (gektin) [inline database/seed.js:20]: Perhaps I'm misunderstanding the exercise, but the way I read this code, every time the seedGen function is invoked (as it is on line 14 of database/index.js), new data will be generated and then added to the database. Is the intent to stabilize that data at any time? 

Also, every time this runs, it is trying to force bundleIds 1 - 100 into the database. What happens if the database has those bundleIds already and collisions occur?

File: package.json [hunk 5]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "navbar-mark",
+  "version": "1.0.0",
+  "description": "BundleNav for FEC",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
```
Comment (gektin) [inline package.json:7]: Before this can be merged in, there will need to be some tests added here.

File: database/seed.js [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+const seed = () => {
+  return db.navbarSave(seedGen());
+}
+
+seedGen = function () {
+  var bundleId, countDown;
+  var timerData = [];
+  for (let i = 1; i < 100; i++) {
+    bundleId = i;
+    timer = Number(Math.floor (Math.random() * 13) + 1);
```
Comment (gektin) [inline database/seed.js:12]: Same consistency point here as in the index.js file in this directory just between `Number(...)` and `Math.floor (...)`.

File: database/seed.js [hunk 7]
```diff
@@ -0,0 +1,24 @@
+const db = require('./index.js');
+
+const seed = () => {
+  return db.navbarSave(seedGen());
+}
```
Comment (gektin) [inline database/seed.js:5]: What is the purpose of this seed function? It isn't invoked anywhere in the code, and its operation duplicates what is already happening in the database/index.js file.

**Review by gektin — CHANGES_REQUESTED** [review]
My major takeaways here are the absence of tests, which will definitely be necessary before we can merge this code, and some confusion about the seed data itself that will be generated. As I mentioned in the internal comments, this may be me misunderstanding the exercise, but I thought we wanted the seed data to be consistent from machine to machine when running the seed file. Because your seeding files invoke `Math.random` in generating the seed data each time the seed script is run, the data will be different every time. 

My other comments are mostly pointing out some extraneous code as well as some style nits. On the whole the nits center around the use of ES6 consistently throughout the code as well as consistency in whitespacing.

Let me know if you have any thoughts on the above.

File: database/seed.js (hunk 6 above)
Comment (gektin) [reply database/seed.js:12]: Of course on my screen the line break in the comment renders it useless, but note the space on line 12 after `Math.floor` is not present after the `Number` and `Math.random` functions are invoked.

File: database/seed.js (hunk 4 above)
Comment (theofficetroll) [reply database/seed.js:20]: Yeah, I left out the drop table.
Leaving as random at this point until we know if instances should be identical.

File: database/seed.js (hunk 7 above)
Comment (theofficetroll) [reply database/seed.js:5]: Legacy code, forgot to remove when I changed the way I was invoking things.

File: package.json (hunk 5 above)
Comment (theofficetroll) [reply package.json:7]: Will be adding tests later, not required at this point. Script was autogenerated when initialized project. Removed script for now.

**Comment by theofficetroll** [comment]
Cleaned up code for consistency. Removed unused script in package.json. Added drop to db.once.

**Pull request closed without merging**